### PR TITLE
feat: chart and dashboard card list view e2e cypress test

### PR DESCRIPTION
**Pull request by lilykuang**
### SUMMARY

- added cypress tests for card view for chart and dashboard list

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: .github/workflows/superset-e2e.yml [hunk 1]
```diff
@@ -12,9 +12,9 @@ jobs:
         browser: ['chrome']
     env:
       FLASK_ENV: development
+      ENABLE_REACT_CRUD_VIEWS: true
```
Comment (lilykuang) [inline .github/workflows/superset-e2e.yml:15]: `ENABLE_REACT_CRUD_VIEWS` is used for card CRUD view

File: superset/config.py [hunk 2]
```diff
@@ -837,7 +837,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # Enables the replacement react views for all the FAB views (list, edit, show) with
 # designs introduced in SIP-34: https://github.com/apache/incubator-superset/issues/8976
 # This is a work in progress so not all features available in FAB have been implemented
-ENABLE_REACT_CRUD_VIEWS = False
+ENABLE_REACT_CRUD_VIEWS = os.environ.get("ENABLE_REACT_CRUD_VIEWS") or False
```
Comment (lilykuang) [inline superset/config.py:840]: This allows `ENABLE_REACT_CRUD_VIEWS` to be editable in e2e yaml file

Comment (nytai) [reply superset/config.py:840]: Should this be on the actual config.py or does it make more sense in the test config?

File: superset-frontend/cypress-base/cypress/integration/chart_list/card_view.test.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    cy.visit(CHART_LIST);
+  });
+
+  it('should load cards', () => {
+    cy.get('.chart-list-view');
+    cy.get('.ant-card').should('be.visible');
+    cy.get('.ant-card').should('have.length', 25);
+  });
+
+  it('should allow favor/unfavor chart card', () => {
```
Comment (willbarrett) [inline superset-frontend/cypress-base/cypress/integration/chart_list/card_view.test.ts:34]: description nit: "favorite/unfavorite" - favor is either bias or a gift at a party.

**Review by willbarrett — COMMENTED** [review]
+1 to Tai's question on the config tweak, one language nit. Thanks for this!

**Review by nytai — APPROVED** [review]
lgtm!

**Comment by willbarrett** [comment]
LGTM2!

**Pull request merged**